libnetwork/network.go

```diff
@@ -1472,64 +1471,6 @@ func (n *Network) deleteSvcRecords(eID, name, serviceID string, epIPv4, epIPv6 n
 	}
 }
 
-func (n *Network) getSvcRecords(ep *Endpoint) []etchosts.Record {
```

Comment:
Interesting; I got curious "so don't we use these records anymore?" but it looks like we _do_, just don't use an accessor for them, and access the field in each function that uses them;

```bash
git grep '.svcRecords'
libnetwork/controller.go:       svcRecords       map[string]*svcInfo
libnetwork/controller.go:               svcRecords:       make(map[string]*svcInfo),
libnetwork/libnetwork_internal_test.go: c.svcRecords[n.ID()] = sr
libnetwork/network.go:  sr, ok := c.svcRecords[networkID]
libnetwork/network.go:          c.svcRecords[networkID] = sr
libnetwork/network.go:  sr, ok := c.svcRecords[networkID]
libnetwork/network.go:  sr, ok := c.svcRecords[networkID]
libnetwork/network.go:  sr, ok := c.svcRecords[networkID]
libnetwork/network.go:  sr, ok := c.svcRecords[networkID]
libnetwork/network.go:  sr, ok := c.svcRecords[networkID]
libnetwork/service_common.go:           c.svcRecords = make(map[string]*svcInfo)
libnetwork/service_common.go:   delete(c.svcRecords, cleanupNID)
libnetwork/service_common_unix_test.go: if len(c.svcRecords) != 2 {
libnetwork/service_common_unix_test.go:         t.Fatalf("Service record not added correctly:%v", c.svcRecords)
libnetwork/service_common_unix_test.go: if len(c.svcRecords) != 1 {
libnetwork/service_common_unix_test.go:         t.Fatalf("Service record not cleaned correctly:%v", c.svcRecords)
libnetwork/service_common_unix_test.go: if len(c.svcRecords) != 0 {
libnetwork/service_common_unix_test.go:         t.Fatalf("Service record not cleaned correctly:%v", c.svcRecords)
```